gluoncv/model_zoo/yolo/darknet.py

```diff
@@ -78,7 +78,7 @@ class DarknetV3(gluon.HybridBlock):
     """
     def __init__(self, layers, channels, classes=1000,
                  norm_layer=BatchNorm, norm_kwargs=None, **kwargs):
-        super(DarknetV3, self).__init__(**kwargs)
+        super(DarknetV3, self).__init__(kwargs.get('prefix', None), kwargs.get('params', None))
```

Comment:
This will allow users to pass unused arguments without warning which is not ideal.
I would recommend iterating the kwargs and raise a warnining if it's not {'params', or 'prefix'}

Reply:
However, in the parent function ` yolo3_darknet53_coco ` for example, there are more than only one place that uses `**kwargs`

https://github.com/dmlc/gluon-cv/blob/a23aef5cb0ce3176e97dd56c550a61f5e844c0b7/gluoncv/model_zoo/yolo/yolo3.py#L631-L639

If unused arguments do pass into  `super(DarknetV3, self).__init__()`, how can you know they'are not used in `get_yolov3()`?

@hetong007

Reply:
whatever passed into Darknetv3 is consumed by its parent class, nothing else, so anything else exists is not desired